test/functional/wallet_sendall.py

```diff
@@ -394,11 +394,48 @@ def sendall_spends_unconfirmed_inputs_if_specified(self):
         self.log.info("Test that sendall spends specified unconfirmed inputs")
         self.def_wallet.sendtoaddress(self.wallet.getnewaddress(), 17)
         assert_greater_than(self.wallet.getbalances()["mine"]["untrusted_pending"], 0)
+        self.wallet.syncwithvalidationinterfacequeue()
```

Comment:
In e73a56f13:

This change should be in the first commit, not here. And the sync call should be done prior to calling `getbalances()` too (one line above the current one).

Reply:
Done